File: default_output/phoenix/person/person_migration.ex
defmodule FinalLocallity.Repo.Migrations.CreatePerson do
  use Ecto.Migration

  def change do
    create table(:person) do
      add :person_uuid, :string
      add :name, :string
      add :age, :integer

      timestamps()
    end

  end
end

File: default_output/flutter/person/person_migration.dart
class Database
{
    void execute(String s) {}
}

class PersonMigration
{
    void createDatabase(Database db)
    {
        db.execute("CREATE TABLE Person"
              +"person_uuid TEXT, "
              +"name TEXT, "
              +"age INTEGER"
            ")"
        );
    }
}
